kernel/aster-nix/src/syscall/epoll.rs

```diff
@@ -109,9 +109,14 @@ fn do_epoll_wait(epfd: FileDesc, max_events: i32, timeout: i32) -> Result<Vec<Ep
         .get_file(epfd)?
         .downcast_ref::<EpollFile>()
         .ok_or(Error::with_message(Errno::EINVAL, "not epoll file"))?;
-    let epoll_events = epoll_file.wait(max_events, timeout.as_ref())?;
+    let result = epoll_file.wait(max_events, timeout.as_ref());
 
-    Ok(epoll_events)
+    if let Err(err) = result {
+        if err.error() == Errno::ETIME {
+            return Ok(Vec::new());
```

Comment:
Could you explain why the epoll system call should not treat `Err(ETIME)` from `EpollFile::wait` as an error? Add the explanation as a comment.